tests/02-flawfinder.sh

```diff
@@ -24,7 +24,11 @@
 
 test_init "Running Flawfinder"
 
```

Comment:
That makes a lot of sense here especially because the `test_init` function does more then just print the line, but it also initializes the space. I agree. @jmstover, pretty please? :)